triggers/tests/daml/MaxInboundMessageTest.daml

```diff
@@ -17,22 +17,11 @@ maxInboundMessageSizeTrigger = Trigger
   }
 
 maxInboundMessageSizeRule : Party -> TriggerA () ()
-maxInboundMessageSizeRule party = query @MessageSize >>= \case
-  [] ->
-      void $ emitCommands [createAndExerciseCmd (MessageSize { p = party }) (CreateN {n = 50000})] []
-  _ -> pure ()
+maxInboundMessageSizeRule party = do
+  void $ emitCommands [createCmd (T party)] []
 
-template MessageSize
+template T
   with
     p : Party
   where
     signatory p
```

Comment:
I assume you are changing the test because you actually do not need to be so complicate. I am right ?

Reply:
first it doesn’t need to be so complex, second sandbox-next is significantly slower so the previous code actually results in timeouts.